pkg/cdi/cache_test.go

```diff
@@ -770,10 +769,10 @@ devices:
 				for {
 					select {
 					case _ = <-stopCh:
-						go syscall.Sync()
+						osSync()
```

Comment:
Minor question: could this be `go osSync()` instead since go routines themselves are not OS-specific?

Reply:
Okay. Move goroutine invocation to this end of the function call.